lex_rule3.py returned with 1

Traceback (most recent call last):
  File "/home/herczy/devel/ply/test/lex_rule3.py", line 25, in <module>
    lex.lex()
  File "/home/herczy/devel/ply/ply/lex.py", line 899, in lex
    raise SyntaxError("Can't build lexer")
SyntaxError: Can't build lexer

ERROR: /home/herczy/devel/ply/test/lex_rule3.py:18: Rule 't_NUMBER' has too many arguments
